README.md

```diff
@@ -20,37 +20,20 @@ Please join the discussion on the Angle Community [Discord Server](https://disco
 
 ## Introduction to Angle
 
-### 🏅 Over-Collateralized, Decentralized, and Capital-Efficient Stablecoin Protocol
+Angle is a decentralized stablecoin protocol composed of smart contracts mostly deployed on the Ethereum blockchain. 
 
-Angle is the first over-collateralized, decentralized and capital-efficient stablecoin protocol. It offers full convertibility between stable assets and collateral at oracle value: you can swap 1 of collateral against 1 of stablecoin and conversely.
+It can be used to issue stablecoins, called agTokens, that are pegged to a specific value. This means that these tokens are designed to mirror the value of the asset they are pegged to. For example, the agEUR is the EUR-pegged stablecoin issued through Angle Protocol. 
 
-This makes the protocol both capital efficient and highly liquid.
+Angle has two modules, or set of smart contracts, from which agTokens can be issued or minted: the original [Angle Core module](/concepts/overview.md) that was released in November 2021, and the new [Angle Borrowing module](/new-module/README.md). 
+
+Angle also relies on a [Governance module](/governance/angle-dao.md) to manage the protocol. 
 
 ### 🛣️ Roadmap
 
-Angle Protocol could be used to issue any stablecoin. and has started on mainnet with agEUR, a Euro stablecoin. Besides creating the first liquid Euro stablecoin, the goal of Angle is to create stablecoins for other types of assets.
+Angle Protocol could be used to issue any stablecoin. and has started on mainnet with agEUR, a Euro stablecoin. Besides creating the first liquid and decentralized Euro stablecoin, the goal of Angle is to create stablecoins for other types of assets.
```

Comment:
the first liquid decentralized (there were other decentralized stablecoins before)